src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -38,6 +38,9 @@
  *******************************************************************************
  ******************************************************************************/
 
+#ifndef SILABS_DOOR_LOCK_SERVER_H
```

Comment:
nit: consider `#pragma once` instead (I am trying to convert the entire codebase to that since it is simpler and faster to comile even (even if marginally so))